client/src/components/Pages/AdminDashboard/BookingsTable.js

```diff
@@ -0,0 +1,268 @@
+    },
+    {
+      title: 'Length of Stay',
+      dataIndex: 'lengthOfStay',
+      key: 'lengthOfStay',
+      ...getColumnSearchProps('lengthOfStay'),
+      sorter: (a, b) =>
+        lengthOfStay(a.startDate, a.endDate) -
+        lengthOfStay(b.startDate, b.endDate),
+      //   sorter: (a, b) =>
+      //     moment(a.startDate || 0).valueOf() - moment(b.startDate || 0).valueOf(),
+      render: (text, record) => (
+        // <Highlighter
+        //   highlightStyle={{ backgroundColor: colors.yellow, padding: 0 }}
+        //   searchWords={[highlightVal]}
+        //   autoEscape
+        //   textToHighlight={moment(text).format('DD/MM/YYYY')}
+        // />
+        <>{lengthOfStay(record.startDate, record.endDate)} days</>
+      ),
+    },
+    {
+      title: 'Paid by Organisation',
+      dataIndex: 'coupon',
+      key: 'coupon',
+      ...getColumnSearchProps('coupon'),
+      render: coupon =>
+        coupon ? (
+          <S.Discount>
+            <PS>{coupon.discountRate}%</PS>
+            <PXS>{coupon.Organisation}</PXS>
+          </S.Discount>
+        ) : (
+          <div>N/A</div>
+        ),
+    },
+    {
+      title: 'Status',
+      dataIndex: 'status',
+      key: 'status',
+      ...getColumnSearchProps('status'),
+      render: status => (
+        <Tag color={tagColors[status]} key={status}>
+          <Highlighter
+            highlightStyle={{ backgroundColor: colors.yellow, padding: 0 }}
+            searchWords={[highlightVal]}
+            autoEscape
+            textToHighlight={status ? status.toUpperCase() : ''}
+          />
+        </Tag>
+      ),
+      filters: [
+        {
+          text: 'pending',
+          value: 'pending',
+        },
+        {
+          text: 'accepted',
+          value: 'accepted',
+        },
+      ],
+      onFilter: (value, record) => record.status.indexOf(value) === 0,
+    },
+    {
+      title: 'Actions',
+      dataIndex: 'action',
+      key: 'actions',
+      render: (text, record) => renderActions(record.status, record),
+    },
+  ];
+  return (
+    <>
+      <Table
+        columns={columns}
+        dataSource={data}
+        pagination={{ pageSize: 5 }}
+        scroll={{ x: '100%' }}
+        loading={loading}
+        expandable={{
+          expandedRowRender: (record, index) => (
+            <S.BookingDetails>
+              <H7C mb="2">Internship Details</H7C>
+              <S.BookingDetailsRow>
+                <S.DetailsItem>
+                  <PXSBold>Organisation</PXSBold>
+                  <PXS>{record.intern.internship.Organisation}</PXS>
+                </S.DetailsItem>
+                <S.DetailsItem>
+                  <PXSBold>Contact Name</PXSBold>
+                  <PXS>{record.intern.internship['Contact Details'].name}</PXS>
+                </S.DetailsItem>
+                <S.DetailsItem>
+                  <PXSBold>Contact Email</PXSBold>
+                  <PXS>{record.intern.internship['Contact Details'].email}</PXS>
```

Comment:
```suggestion
                  <a href=`mailto:${value}`><PXS>{record.intern.internship['Contact Details'].email}</PXS></a>
```
or something like that, Also the whole expandable section need a little bit of styling (spacing and stuff)
this is how it looks when I open it
![image](https://user-images.githubusercontent.com/18149438/79062793-db5d6300-7ca5-11ea-8cf9-2a871448e8bc.png)

Reply:
done